docs/user-guide/command-line-interface.md

```diff
@@ -361,6 +362,16 @@ This option instructs ESLint to try to fix as many issues as possible. The fixes
 1. This option throws an error when code is piped to ESLint.
 1. This option has no effect on code that uses processors.
 
+If you want to fix code from stdin or otherwise want to get the fixes without actually writing them to the file, use the [`--fix-dry-run`](--fix-dry-run) option.
+
+#### `--fix-dry-run`
+
+This option has the same effect as `--fix` with one difference: the fixes are not being saved to the file system. This makes fixing the code from stdin possible.
```

Comment:
Nitpick: I think the word "being" should be removed from the first sentence ("the fixes are not saved to the file system.")